test/test_board.h

```diff
@@ -122,9 +122,10 @@ class testBoard : public Board
   void led1_toggle() override;
 
 //Backup memory
-  bool has_backup_data() override;
-  BackupData get_backup_data() override;
-
+  void backup_memory_init() override {}
+  bool backup_memory_read(void *dest, size_t len) override { (void)dest; (void)len; return false; }
+  void backup_memory_write(const void *src, size_t len) override { (void)src; (void)len; }
+  void backup_memory_clear(size_t len) override { (void)len; }
```

Comment:
It might be good to add some tests for such things as not accidentally rearming, but that could perhaps wait for a future PR.

Reply:
I think that's a pretty important thing to do.

#381